.github/workflows/UnitTests.yml

```diff
@@ -99,6 +99,10 @@ jobs:
       run: |
         docker run -v /home/runner/actions-runner/_work/maxtext/maxtext:/app --rm --privileged maxtext_base_image bash -c \
         'python3 MaxText/train.py MaxText/configs/base.yml run_name=runner_$(date +%Y-%m-%d-%H-%M)-${RANDOM} base_output_directory=gs://runner-maxtext-logs dataset_path=gs://maxtext-dataset quantization=int8 steps=2 enable_checkpointing=false'
+    - name: Test fp8_training
```

Comment:
We might wanna add this to the GPU tests below, but we can do that in the follow-up decode PR :). This should do a good job covering the functionality added!